### PR TITLE
Astronomer tap optional if cli install fails

### PR DESCRIPTION
**Pull request by abhishekbhakat**
Installing a specific version for the first time may not work without astronomer tap available.

We can check if `astronomer/tap` if is in the list when you try command `brew tap` ? If it's not there, then add that with `brew tap astronomer/tap` and then try for example `brew install astro@1.18.2`

### CODE REVIEW

**Comment by abhishekbhakat** [comment]
Screenshot 2023-10-17 at 06 44 47
Renders nicely 👌

Maybe we can make the step 4 command as:
```
brew install astro@<major.minor.patch-version>
```

File: astro/cli/install-cli.md [hunk 1]
```diff
@@ -71,13 +71,29 @@ astro version
...
+
+  ```sh
+  brew tap astronomer/tap
+  ```
+
+4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version by adding the `astro@<major.minor.patch-version>` flag to the following command:
 
-2. Install the Astro CLI. See [Installation](cli/install-cli.md?tab=mac#installation). 
+   ```sh
+   brew install astro
```
Comment (lzdanski) [inline astro/cli/install-cli.md:95]:
```suggestion
    brew install astro@<major.minor.patch-version>
```

**Review by lzdanski — APPROVED** [review]

File: astro/cli/install-cli.md [hunk 2]
```diff
@@ -71,13 +71,29 @@ astro version
...
+   ```
+
+3. If updating Homebrew doesn't work and this is the first time you're installing the CLI, check to see if `astronomer/tap` is in your [Homebrew Tap](https://docs.brew.sh/Taps) list: 
+
+  ```sh
+  brew tap astronomer/tap
+  ```
+
+4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version by adding the `astro@<major.minor.patch-version>` flag to the following command:
```
Comment (lzdanski) [inline astro/cli/install-cli.md:92]:
```suggestion
4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version, you can add the `astro@<major.minor.patch-version>` flag to the `brew install` command:
```

File: astro/cli/install-cli.md [hunk 3]
```diff
@@ -71,13 +71,29 @@ astro version
...
+   brew install astro
+   ```
+
+3. If updating Homebrew doesn't work and this is the first time you're installing the CLI, check to see if `astronomer/tap` is in your [Homebrew Tap](https://docs.brew.sh/Taps) list: 
+
+  ```sh
+  brew tap astronomer/tap
+  ```
+
+4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version, you can add the `astro@<major.minor.patch-version>` flag to the `brew install` command:
```
Comment (lzdanski) [inline astro/cli/install-cli.md:92]:
```suggestion
4. Install the Astronomer CLI. You can choose to specify the major and minor version by adding the `astro@<major.minor.patch-version>` flag to the `brew install` command:
```

**Comment by lzdanski** [comment]
This PR has been superseded by https://github.com/astronomer/docs/pull/2957

Found that netlify preview builds don't occur with PR's submitted from forks. I created a PR that fixes the contributing doc with this clarification and added the changes from this PR to it. Closing PR.

**Pull request closed without merging**